http_image_provider/lib/consolidate_response.dart

```diff
@@ -0,0 +1,106 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:http/http.dart';
+
+export 'dart:io' show HttpClientResponse;
```

Comment:
This unfortunately breaks this package compatibility on web. You'll need to use conditional imports to make this happen, or get rid of `dart:io` imports

Reply:
Done. Removed from http_image_provider.dart also.

Reply:
Awesome thanks!